Thai duong

thaiduongcai opened the pull request:
Before issuing a pull request, please see the contributing page.

thaiduongcai committed:
LinearSlide with PIDController
build.gradle update

thaiduongcai committed:
update TeleOp

thaiduongcai committed:
update LinearSlidePID

sumirolabs commented on TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TestPrograms/LinearSlidePID.java:
```diff
@@ -0,0 +1,108 @@
+    @Override
+    public void loop() {
+
+        dashboard.sendTelemetryPacket(packet);
```
Since you send this packet on line 81, I don't think you need it here.

sumirolabs commented on TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TestPrograms/PIDFLinearSlide.java:
```diff
@@ -0,0 +1,95 @@
+            double leftEncoder = leftSlide.getCurrentPosition();
+            double rightEncoder = rightSlide.getCurrentPosition();
+
+            double power = pid.calculate(leftEncoder, targetPosition);
```
Shouldn't you be setting the `power` **_after_** adjusting the target position (lines 55 & 60 below?

thaiduongcai committed:
RightSideAutonomous - Path 4 Specimen 80%

thaiduongcai committed:
RightSideAutonomous With Spline

thaiduongcai committed:
Update build.gradle

thaiduongcai committed:
RightSideAutonomous - 12/18/2024

thaiduongcai committed:
New 4 Specimens Pathing && Add action for armslidesclaw && Parallel Action

thaiduongcai committed:
add traj in teleop

made it more auto-like

thaiduongcai committed:
Preset for Teleop + 2 specimen autonomous finished 80%

thaiduongcai committed:
Finished Autonomous 2 Specimen

thaiduongcai committed:
Finished TeleOP (Placing Sample Error)

thaiduongcai committed:
finished teleop - finish auto except wrist

thaiduongcai committed:
organized code + finished up teleop and auto

Still have some problem with wrist and arm

Going to make the old method for autonomous to pick up the specimen

thaiduongcai committed:
Same summary with the previous push

same

thaiduongcai committed:
Perfect Teleop + 4 Specimen auto 100%

thaiduongcai committed:
Perfect TeleOP + 4 Specimen 100%

If there is problem with running code, just need to delete ROBOT.java, MainDrive.java and Subsystem Test folder

thaiduongcai committed:
InspiredAward Push GOOD LUCKS GUYSSS!!

thaiduongcai committed:
PLEASE LAST TIME

thaiduongcai committed:
RIGHTSIDEAUTONOMOUS FINISHED + LEFTSIDE FINISHED NEED TO CHANGE A BIT

thaiduongcai committed:
CHANGE AUTONOMOUS PATHING AND SLIDE EXTENSION

thaiduongcai committed:
teleop + right auto

thaiduongcai committed:
JADON REQUESTED

thaiduongcai committed:
RightSide + LeftSide

- Rightside: new path, faster, more precise
- LeftSide: Improved

thaiduongcai committed:
LeftSide working perfectly + TELEOP WORKING IN THE COMP

thaiduongcai committed:
LeftSide + RightSide new pathing + Camera

thaiduongcai committed:
Roadrunner Updated + FTCSDK Updated

Everything is up-to-date now!

thaiduongcai committed:
Camera + RightSideAuto

thaiduongcai committed:
Return TeamCode

Roadrunner ready for tuning
Value of wrist all set!

thaiduongcai committed:
camera

thaiduongcai committed:
Camera moving horizontally, and slides move to the game piece

thaiduongcai committed:
Autonomous + Camera working 80% wrap the piece

thaiduongcai committed:
Angle needs to be change

thaiduongcai committed:
Camera working perfectly + need to tune a bit

thaiduongcai committed:
New Alignment with regression

thaiduongcai committed:
LEFTSIDE AUTO

thaiduongcai committed:
pushed

thaiduongcai committed:
NEWEST PUSH

thaiduongcai committed:
6 SAMPLES WITHOUT TIME

thaiduongcai committed:
6 SAMPLES SO FAST AND FIX TELEOP

thaiduongcai committed:
1/3/2025 - 9:24PM

thaiduongcai committed:
hopefully this is not last push
